operational/fnms/schedule_fnms_reports/README.md

```diff
@@ -1,46 +1,66 @@
-## Schedule FlexNet Manager report
+# Schedule FlexNet Manager report
+
+## What it does
 
-### What it does
 This policy will run a FlexNet Manager report (Custom view) and send the result via email.
 The policy is a report only policy, no action is taken during the Policy Escalation.
 
 The report / Mail output looks like this:
 ![Alt text][emailoutput]
 
 Current limitations:
-- This only works with the FNMS Cloud
+
+- This only works with the FNMS Cloud or a onPrem installation available on the internet.
```

Comment:
Are there other services or should this line be removed since these are no longer limitations.